homeassistant/components/google_assistant/trait.py

```diff
@@ -649,7 +649,7 @@ def sync_attributes(self):
 
     def query_attributes(self):
         """Return EnergyStorage query attributes."""
-        battery_level = self.state.attributes.get(ATTR_BATTERY_LEVEL)
+        battery_level = self.state.attributes.get(ATTR_BATTERY_LEVEL) or 0
```

Comment:
Let's not assume 0, instead, we need to make sure the if/else tree below only triggers when the value is not `None`.

Reply:
I am not sure if we can return {}
```
        if battery_level is None:
            return {}
        elif battery_level == 100:
```